spec/mongoid/collection_configurable_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+class CollectionConfigurableCapped
+  include Mongoid::Document
+
+  store_in collection_options: {
+    capped: true,
+    size: 10000
+  }
+end
+
+class CollectionConfigurableUnknownOptions
+  include Mongoid::Document
+
+  store_in collection_options: {
+    i_am_not_an_option: true,
+    let_me: { raise: true }
+  }
+end
+
+describe Mongoid::CollectionConfigurable do
+  before(:each) do
+    [
+      CollectionConfigurableCapped,
+      CollectionConfigurableUnknownOptions
+    ].each do |klaz|
+      klaz.collection.drop
+    end
+  end
+
+  context 'when collection does not exist' do
+    context 'with an unknown collection options' do
+      let(:subject) do
+        CollectionConfigurableUnknownOptions
+      end
+
+      it 'creates the collection' do
+        subject.create_collection
+        expect(
+          subject.collection.database.list_collections(filter: { name: subject.collection_name.to_s })
+        ).not_to be_empty
+      end
+    end
+
+    context 'with supported options' do
+      let(:subject) do
+        CollectionConfigurableCapped
+      end
+
+      it 'creates the collection' do
```

Comment:
maybe add an `it` that calls `collstats` and makes sure the options were actually set?

Reply:
Not sure if we should have the driver functionality here, but some additional coverage won't hurt. Added the requested.